init/wwivd_ui.cpp

```diff
@@ -21,7 +21,7 @@
 #include <cstdint>
 #include <memory>
 #include <string>
-
+#include <math.h>
```

Comment:
Did this cause a compile error?

usually including C only headers we do as:

#include <cmath>